client/dashboard/components/section-header/style.scss

```diff
@@ -5,6 +5,16 @@
 	position: relative;
 	z-index: 1;
 
+	.components-button.dashboard-page-header__back-button {
```

Comment:
Ideally we should avoid CSS overrides like this. I think it's okay for now given that this is a temporary fix though,